x-pack/packages/kbn-elastic-assistant/impl/assistant/assistant_overlay/index.tsx

```diff
@@ -79,7 +78,7 @@ export const AssistantOverlay: React.FC = React.memo(() => {
   return (
     <>
       {isModalVisible && (
-        <StyledEuiModal onClose={handleCloseModal}>
+        <StyledEuiModal onClose={handleCloseModal} data-test-subj="ai-assistant-modal">
```

Comment:
this isn't used anywhere in the repo, did you intent to use it or was that just for temporary testing and should be removed?

Reply:
yes, there is not a ton of test coverage (any?), but after all this bug fixing I plan to come back with [a unit test PR](https://github.com/elastic/security-team/issues/6989). I left this in there both because I plan on coming back to use it, and because it helps debug as I can `cmd + F` and search for this label to find the markup when working with Chrome DevTools.